eos/utils/kmatrix_TEST.cc

```diff
@@ -26,63 +27,64 @@
 using namespace test;
 using namespace eos;
 
-///////////////////////
+/////////////////////////////////////////////
 /// Simplest K matrix, 1 channel, 1 resonance
-///////////////////////
+/////////////////////////////////////////////
 struct PPchan11 :
     public KMatrix<1,1>::Channel
```

Comment:
I think we need to talk about this. I would probably have used a template for the ``PPChannel``, i.e.,
```
template <unsigned nchannels_, unsigned nresonances_>
struct PPChannel :
    public KMatrix<nchannels_, nresonances_>::Channel
{
    // ...
};
```

Reply:
I don't think we need to talk >< of course it needs to be a templated `struct`, but when I started writing this (i.e. 3 years ago ><) I was not super familiar in C++...